src/charts/scale.js

```diff
@@ -103,7 +103,7 @@ export default function Scale () {
   function buildMeasureNameLookup () {
     return d => {
       if (d === "x") {
-        return data.dataBySeries[0].dimensionName
+        return data.dataBySeries[0] ? data.dataBySeries[0].dimensionName : ""
```

Comment:
IMO it's not a hack, it's a proper protection. So I accept it, but does it work if you do:
```
data.dataBySeries[0] && data.dataBySeries[0].dimensionName
```
It would protect the array but also return undefined instead of an empty string, which is more consistent with the other cases. Consistently returning undefined makes it easier to check for the absence of a measure name in the data.